cmd/atlas/internal/cmdapi/migrate.go

```diff
@@ -429,16 +429,23 @@ func CmdMigrateDiffRun(cmd *cobra.Command, args []string) error {
 	}
 	opts := []migrate.PlannerOption{migrate.PlanFormat(f)}
 	if dev.URL.Schema != "" {
-		opts = append(opts, migrate.PlanSchema(dev.URL.Schema))
+		// Disable tables qualifier in schema-mode.
+		opts = append(opts, migrate.PlanWithSchemaQualifier(""))
```

Comment:
I think the API is a little weird here. Atlas does plan with qualifier by default, correct? Why not have another option `PlanWithoutSchemaQualifier()` (that maybe call the above)?

Reply:
> Atlas does plan with qualifier by default, correct?

Only in realm mode. 

> PlanWithoutSchemaQualifier

hehe, that's funny, because I started the code this way, but since I want to provide a flag (with default to ""), I decided to keep only one option.

Reply:
oki